tests/pipelines/wan/test_wan_image_to_video.py

```diff
@@ -247,3 +268,32 @@ def get_dummy_inputs(self, device, seed=0):
             "output_type": "pt",
         }
         return inputs
+
+    def test_inference(self):
+        device = "cpu"
+
+        components = self.get_dummy_components()
+        pipe = self.pipeline_class(**components)
+        pipe.to(device)
```

Comment:
Not important for this PR. But at some point, I think we should switch to `torch_device` instead of fixing devices like this. I see downsides and upsides:

* Upside: same code can be used for quickly checking if the `pipe` can be run on different accelerators (depending on what we get for `torch_device`.
* Downside: The `expected_slice` will vary but I guess we can leverage a combination of cosine similarity based checks, enforcement of determinism (like [this](https://github.com/huggingface/diffusers/blob/ef1e6287291a945665b0c43f4863228b8e9131df/src/diffusers/utils/testing_utils.py#L1129)).

No need to fret for this PR. We will get to it later.